UI/GameSettingsScreen.cpp

```diff
@@ -382,7 +382,7 @@ void GameSettingsScreen::CreateViews() {
 	LinearLayout *networkingSettings = new LinearLayout(ORIENT_VERTICAL);
 	networkingSettings->SetSpacing(0);
 	networkingSettingsScroll->Add(networkingSettings);
-	tabHolder->AddTab(s->T("Networking"), networkingSettingsScroll);  // TODO: This string was originally just an item header so in the wrong category
+	tabHolder->AddTab(n->T("Networking"), networkingSettingsScroll);
```

Comment:
I'm not sure to understand, do you mean we should use "ms" instead of "n" for the title ?